### ISSUE TITLE
Use of validation set

### ISSUE THREAD

**Issue by pinarezgicol** [ElementAI/baal#162]
In active learning dataset, it would be nice to provide a validation dataset. Can it be added as a new feature please?

**Comment by Dref360** [comment]
Hello, sorry for the delay! 

We currently support early stopping in `ModelWrapper.train_test_on_dataset`. Is that what you mean? We could rename it to `train_evaluate_on_dataset` to make it clearer!

Usually we can't use early stopping in Active learning as we always overfit in the first few steps. We do have a [post](https://baal.readthedocs.io/en/latest/reports/double_descend.html) on that.

**Comment by parmidaatg** [comment]
@pinarezgicol can we close this issue? Does the renaming help?

**Comment by Dref360** [comment]
Closing
